appinventor/appengine/src/com/google/appinventor/shared/rpc/user/User.java

```diff
@@ -146,7 +146,7 @@ public boolean getUserTosAccepted() {
 
   @Override
   public boolean getIsAdmin() {
-    return isAdmin;
+    return true;
```

Comment:
This doesn't seem correct!

Reply:
Yes, it was for debugging purpose. I will change that